src/System.Windows.Forms/src/System/Windows/Forms/Control.ActiveXImpl.cs

```diff
@@ -529,7 +530,7 @@ internal void Close(OLECLOSE dwSaveOption)
         /// </summary>
         private static byte[] FromBase64WrappedString(string text)
         {
-            if (text.IndexOfAny(new char[] { ' ', '\r', '\n' }) != -1)
+            if (text.IndexOfAny(s_whitespace) != -1)
```

Comment:
Or simply 
```suggestion
            if (text.AsSpan().ContainsAny(' ', '\r', '\n'))
```
though we could use the computed index to slightly optimise the loop below

Reply:
I'm going to go the `SearchValues` route for now. Happy to entertain any other optimization PRs.